config/index.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
     proxyTable: {}
   },
   backend: {
-    host: process.env.BACKEND_HOST || 'kuzzle',
+    host: process.env.BACKEND_HOST || 'localhost',
```

Comment:
Do e2e tests work with this? The name associated to the Kuzzle Proxy is `kuzzle`, in the Docker-compose file.